src/lib.rs

```diff
@@ -46,7 +48,7 @@ use crate::windows as sys;
 
 pub use crate::{
     string::{EitherOsStr, IntoOsString, ToOsStr},
-    sys::{Error, OsStr, OsString},
+    sys::{Error, FileDesc, OsStr, OsString},
```

Comment:
`FileDesc` should be private API-only, I don't think it is a good idea to leak it.